src/frontEnd/vueChecks/createSummary.js

```diff
@@ -1,5 +1,212 @@
+const { eslintMetrics } = require('../../linterChecks/constants');
+const dependenciesMetrics = require('../../dependenciesChecks/constants');
+const seoMetrics = require('../seoChecks/constants');
+
+const limits = {
+  reduxRecomposePercentage: 70,
+  i18nPercentage: 40
+};
+
+const testSummary = (summary, reports) => {
+  summary.push({
+    metric: 'SUMMARY-TESTING-1',
+    description: 'La arquitectura de la aplicación se encuentra preparada para implementar test unitarios',
+    value: reports.some(elem => elem.metric === dependenciesChecks.JEST && elem.value)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-TESTING-2',
+    description:
+      'La arquitectura de la aplicación se encuentra preparada para implementar test de integración',
+    value: 'N/A'
+  });
+
+  summary.push({
+    metric: 'SUMMARY-TESTING-3',
+    description: 'Hay una cobertura del 70% o más en los test unitarios',
+    value: reports.some(
+      elem => elem.metric === testMetrics.CODE_COVERAGE && elem.value >= limits.minTestCoverage
+    )
+  });
+
+  summary.push({
+    metric: 'SUMMARY-TESTING-4',
+    description:
+      'Cada nueva funcionalidad unida a "development" está acompañada por un test de integración que valida el correcto funcionamiento de la misma.',
+    value: 'N/A'
+  });
+
+  summary.push({
+    metric: 'SUMMARY-TESTING-5',
+    description: 'El proyecto usa vue-test-utils',
+    value: reports.some(report => report.metric === dependenciesChecks.VUE_TEST_UTILS && report.value)
+  });
+};
+
+const securitySummary = (summary, reports) => {
+  summary.push({
+    metric: 'SUMMARY-SECURITY-1',
+    description: 'Existe un .env con variables de entorno en el proyecto',
+    value: reports.some(elem => elem.metric === envMetrics.ENV_IS_USED && elem.value)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-SECURITY-2',
+    description:
+      'Las credenciales para firmar a producción se encuentran en un lugar seguro y disponible para el equipo de desarrollo y el TM',
+    value: 'Manual'
+  });
+
+  summary.push({
+    metric: 'SUMMARY-SECURITY-3',
+    description: 'No se guardan claves secretas en texto plano o en constantes dentro de la aplicación',
+    value: 'Manual'
+  });
+
+  summary.push({
+    metric: 'SUMMARY-SECURITY-5',
+    description: 'Las contraseñas no son nunca visibles para el usuario despues del login',
+    value: 'Manual'
+  });
+};
+
+const buildingSummary = (summary, reports) => {
+  summary.push({
+    metric: 'SUMMARY-BUILDING-1',
+    description: 'El proyecto utiliza la ultima o anteultima versión del framework',
+    value: !reports.some(
+      elem => elem.metric === generalMetrics.OUTDATED_DEPENDENCIES && elem.value.includes('vue')
+    )
+  });
+
+  summary.push({
+    metric: 'SUMMARY-BUILDING-2',
+    description: 'Las dependencias del proyecto están actualizadas',
+    value:
+      reports.filter(elem => elem.metric === generalMetrics.UNUSED_DEPENDENCIES).length <=
```

Comment:
Why are you using `filter`? Shouldnt it be `some`??

Reply:
I've doing the same like react's `createSummary`. The answer is because the value of `UNUSED_DEPENDENCIES`'s report its the list of packages unused. Let's check this code snippet from `dependenciesCheck`:

```
  packages.forEach(dependency => {
    const { moduleName, latest, packageJson, unused, bump } = dependency;
    if (unused) {
      dependenciesResults.push({
        metric: dependenciesMetrics.UNUSED_DEPENDENCIES,
        description: 'Dependencia no usada',
        value: moduleName
      });
    } else if (bump && bump !== 'patch') {
      dependenciesResults.push({
        metric: dependenciesMetrics.OUTDATED_DEPENDENCIES,
        description: 'Dependencia no actualizada',
        value: `${moduleName} Version: packageJson: ${packageJson} -> ultima ${latest}`
      });
    }
  });
```